### PR TITLE
magento/devdocs#2957 Adding an example of the "fieldset" node configuration in the definition.xml

### PR DESCRIPTION
**Pull request by DenisSaltanahmedov**
## Purpose of this pull request
This pull request (PR) adds an example of the "fieldset" node configuration in the definition.xml

## Affected DevDocs pages
-  https://devdocs.magento.com/guides/v2.4/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.html

## Links to Magento source code
-  src/guides/v2.4/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.md

## Links to issue
https://github.com/magento/devdocs/issues/2957

### CODE REVIEW

File: src/guides/v2.3/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.md [hunk 1]
```diff
@@ -119,6 +117,20 @@ The default configuration of a UI component is declared in one of the following
 *  in the [`definition.xml` file]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/ui_component/etc/definition.xml)
 *  in both places, in which case the configurations merge (the UI component `.js` file has priority).
 
-In the above example, the Fieldset UI component uses a merged configuration from both the `definition.xml` file and from the UI component's `.js` file.
+In the example, the `<item name="component">...</item>` node within `<fieldset>` is omitted, because this property of the Fieldset UI component is already defined in `definition.xml`.
+
+### Example of the "fieldset" node configuration in the definition.xml
```
Comment (BarnyShergold) [inline src/guides/v2.3/ui_comp_guide/concepts/ui_comp_xmldeclaration_concept.md:122]:
```suggestion
### Example of the `fieldset` node configuration in the `definition.xml`
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
2 small suggestions

**Comment by DenisSaltanahmedov** [comment]
@BarnyShergold thanks for your review. I have added the fix.

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8773`.

**Pull request merged**